redash/settings/__init__.py

```diff
@@ -467,3 +467,6 @@ def email_server_is_configured():
 
 # Email blocked domains, use delimiter comma to separated multiple domains
 BLOCKED_DOMAINS = set_from_string(os.environ.get("REDASH_BLOCKED_DOMAINS", "qq.com"))
+
+# AWS
+REDASH_DATABASE_IAM_AUTH = parse_boolean(os.environ.get("REDASH_DATABASE_IAM_AUTH", "false"))
```

Comment:
From the variable name, I cannot know whether it is AWS's IAM or GCP's IAM.
I think it is better to change the variable name to like "AWS_IAM" instead of "IAM".